launch/launch/launch_context.py

```diff
@@ -171,13 +182,24 @@ def unregister_event_handler(self, event_handler: BaseEventHandler) -> None:
 
     def emit_event_sync(self, event: Event) -> None:
         """Emit an event synchronously."""
-        self.__logger.debug("emitting event synchronously: '{}'".format(event.name))
```

Comment:
Why remove this debug line?

Reply:
It was not intentional. Added back 098b56a